### PR TITLE
add chrony

### PR DESCRIPTION
**Pull request by developer-guy**
Fixes:

Related:

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
  - [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [x] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

#### For version bump PRs

- [ ] The `epoch` field is reset to 0
- [ ] Patch source: _patch source here_

### CODE REVIEW

File: chrony.yaml [hunk 1]
```diff
@@ -0,0 +1,97 @@
+# Generated from https://git.alpinelinux.org/aports/plain/main/libxshmfence/APKBUILD
```
Comment (ajayk) [inline chrony.yaml:1]: probably reference to libxshmfence needs to be changed ?

File: chrony/chrony.pre-install
Comment (tuananh) [inline chrony/chrony.pre-install:1]: we dont need this hook i suppose?

Comment (developer-guy) [reply chrony/chrony.pre-install:1]: we need this to create `chrony` user and group where we use `chown` command.

File: chrony.yaml [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+
+      mkdir -p "${{targets.destdir}}"/var/lib/chrony \
+           "${{targets.destdir}}"/etc/chrony
+
+      # chrony.drift must be writable by chrony user
+      chown ${{package.name}}:${{package.name}} "${{targets.destdir}}"/etc/chrony
+      install -m644 ./chrony.conf "${{targets.destdir}}"/etc/chrony/chrony.conf
+      chown root:root "${{targets.destdir}}"/etc/chrony/*
+      chown ${{package.name}}:${{package.name}} "${{targets.destdir}}"/var/lib/chrony
+      touch "${{targets.destdir}}"/var/lib/chrony/chrony.drift
```
[the remark this answers is not in the document]
Comment (developer-guy) [reply chrony.yaml:74]: I think we are not going to create an image for chrony as the Xage customer only requested for apk, [here](https://github.com/chainguard-dev/customer-issues/issues/429), @amdawson can correct me?

**Review by imjasonh — DISMISSED** [review]

File: chrony.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      - busybox
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha512: 1394bac3ed684352fe89b7fef7da50e61f9f522abee807627ae1fc4c2dde891017bc8e5b13759fced028f3a1e875d5e4e5a4f85de65c63b5f83d0ca03bb4c5df
+      uri: "https://download.tuxfamily.org/chrony/chrony-${{package.version}}.tar.gz"
+
+  - runs: |
+      ./chrony/chrony.pre-install
```
Comment (kaniini) [inline chrony.yaml]: Why?  Define the `chrony` user as part of the build environment definition instead.

**Review by kaniini — CHANGES_REQUESTED** [review]

**Review by kaniini — APPROVED** [review]

**Pull request merged**